pkg/lambdaworks/lambdaworks.go

```diff
@@ -171,6 +171,15 @@ func (a Felt) Div(b Felt) Felt {
 	return fromC(result)
 }
 
+// Returns the felt
+func (f Felt) ToString() string {
+	var f_c = f.toC()
+	resultPtr := C.to_signed_felt(&f_c[0])
+	defer C.free_string(resultPtr)
+	goResult := C.GoString(resultPtr)
+	return goResult
+}
+
```

Comment:
This can be quite misleading, as it is not the standard decimal representation of a Felt,we could rename it to `ToSignedFeltString`

Reply:
Agree!